### PR TITLE
Enable CORS

### PR DESCRIPTION
**Pull request by acabunoc**
* http://enable-cors.org/server_expressjs.html

### CODE REVIEW

**Comment by acabunoc** [comment]
This will let us start working on #72

**Comment by acabunoc** [comment]
@guahanweb - could use a review here if you're familiar w/ express!

File: src/app.js [hunk 1]
```diff
@@ -54,6 +54,12 @@ module.exports = function (badgerService) {
     state: 'none'
   });
 
+  app.use(function (req, res, next) {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
```
Comment (guahanweb) [inline src/app.js]: The only thing to consider is whether there are ever going to be additional headers that may be sent from the client. Things like `Cache-Control` or other browser headers that may be injected?

**Comment by guahanweb** [comment]
cr:gh - LGTM, just consider the header comment. If you're confident with those, I'd say merge it!

File: src/app.js (hunk 1 above)
Comment (acabunoc) [inline src/app.js]: This is a good point - we should def revisit when we add a  CDN.

**Comment by acabunoc** [comment]
Thanks @guahanweb :)

**Pull request merged**